microsoft-teams/1.0/antora.yml

```diff
@@ -0,0 +1,17 @@
+name: microsoft-teams-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
```

Comment:
1.0.0 as well.

Reply:
For documentation purposes, we use only the minor version. This is what appears in the navigation.